bin/pygrb/pycbc_pygrb_page_tables

```diff
@@ -44,173 +46,139 @@ __version__ = pycbc.version.git_verbose_msg
     Returns
     -------
-    inj_data: dict of parameter numpy.array pairs
-        Found or missed injection parameter dictionary.
+    data: tuple of dictionaries
+        Found and missed injection parameter dictionaries.
     """
 
-    inj_data = {}
-
-    # Local copies of variables entering the BestNR definition
-    chisq_index = opts.chisq_index
-    chisq_nhigh = opts.chisq_nhigh
-    null_thresh = list(map(float, opts.null_snr_threshold.split(',')))
-    snr_thresh = opts.snr_threshold
-    sngl_snr_thresh = opts.sngl_snr_threshold
-    new_snr_thresh = opts.newsnr_threshold
-    null_grad_thresh = opts.null_grad_thresh
-    null_grad_val = opts.null_grad_val
-
-    # Get inj params
-    inj_params = ['mchirp', 'mass1', 'mass2', 'distance', 'inclination',
-                  'spin1x', 'spin1y', 'spin1z', 'spin2x', 'spin2y', 'spin2z']
+    inj_data = h5py.File(hdf_file, 'r')
+    inj_params = ['mass1', 'mass2', 'distance', 'inclination', 'ra', 'dec',
+                  'polarization', 'spin1x', 'spin1y', 'spin1z', 'spin2x',
+                  'spin2y', 'spin2z', 'tc']
+    found_data = {}
+    # Missed injections (ones not recovered at all)
+    missed_data = {}
+    logging.info('Loading injections...')
+
+    # Load injections parameters
     for param in inj_params:
-        inj_data[param] = get_column(inj_table, param)
-    inj_data['ra'] = get_column(inj_table, 'longitude')
-    inj_data['dec'] = get_column(inj_table, 'latitude')
-    inj_data['time'] = get_column(inj_table, 'geocent_end_time') \
-                + 1e-9 * get_column(inj_table, 'geocent_end_time_ns')
+        missed_data[param] = inj_data['missed/%s' % param][...]
+        found_data[param] = inj_data['found/%s' % param][...]
+
+    # Calculate injections mchirp
+    missed_data['mchirp'] = mchirp_from_mass1_mass2(missed_data['mass1'],
+                                                    missed_data['mass2'])
+    found_data['mchirp'] = mchirp_from_mass1_mass2(found_data['mass1'],
+                                                   found_data['mass2'])
+
+    # Calculate effective distance for the ifos
+    found_data = additional_injection_data(found_data, ifos)
+    missed_data = additional_injection_data(missed_data, ifos)
+
+    # Get recovered parameters and statistic values for the found injections
+    # Recovered parameters
+    for param in ['mass1', 'mass2', 'spin1z', 'spin2z']:
+        found_data['rec_%s' % param] = np.array(bank_file[param])[inj_data['network/template_id']]
+
+    found_data['end_time_gc'] = \
+        inj_data['network/end_time_gc'][...]  # needed to use bestNR
+    found_data['rec_mchirp'] = mchirp_from_mass1_mass2(
+        found_data['rec_mass1'],
+        found_data['rec_mass2'])
+    # Recover RA and Dec from the trig file
+    trig_id = trig_data['network/template_id'][...]
+    event_id_found = np.array([ np.where(trig_id == found_id)[0][0] 
+                               for found_id in inj_data['network/template_id'][...]])
```

Comment:
```suggestion
    # Recovered RA and Dec
    found_data['rec_ra'] = np.degrees(np.array(inj_data['network/ra']))
    found_data['rec_dec'] = np.degrees(np.array(inj_data['network/dec']))
```

Reply:
The injection data tells you where the injections were placed (e.g., `['missed/ra']`) but also where the ones recovered were searched for and found (e.g., `['network/ra']`).  When it is a single point search, the recovered RA and Dec will contain the same 2 values for all injections.